src/Couchbase.Lite.Shared/Support/Status.cs

```diff
@@ -110,16 +139,21 @@ public static unsafe void ConvertNetworkError(Exception e, C4Error* outError)
                             c4err.domain = C4ErrorDomain.NetworkDomain;
                             c4err.code = (int) C4NetworkErrorCode.TLSHandshakeFailed;
                         }
-                    #if __IOS__
+
+#if __IOS__
                         if (ie.Message == "Connection closed.") {
                         //AppleTlsContext.cs
                         //case SslStatus.ClosedAbort:
                         //  throw new IOException("Connection closed.");
                         message = ie.Message;
-                            c4err.domain = C4ErrorDomain.POSIXDomain;
-                            c4err.code = PosixBase.GetCode(nameof(PosixWindows.ECONNRESET));
+                            c4err.domain = C4ErrorDomain.NetworkDomain;
+                            c4err.code = (int)C4NetworkErrorCode.ConnectionReset;
                         }
-                    #endif
+// #elif __ANDROID__
+// EPIPE = .NET SocketException's SocketErrorCode ConnectionReset
+// Broken pipe is caused by the system sending SIGPIPE signal, which is an inter-process communication mechanism of Linux.
```

Comment:
What is this comment here for?

Reply:
I think it's a good info. Let me move to a more proper location.